numpyro/contrib/module.py

```diff
@@ -374,7 +383,11 @@ def random_haiku_module(
                                       prior={"linear.b": dist.Cauchy(), "linear.w": dist.Normal()},
                                       input_shape=(4,))
 
-    :type prior: dict or ~numpyro.distributions.Distribution
+        Alternatively, we can use a callable. For example the following are equivalent:
+            prior=(lambda name, shape: dist.Cauchy() if name.startswith("b") else dist.Normal()),
+            prior={"bias": dist.Cauchy(), "kernel": dist.Normal()},input_shape=(4,))
```

Comment:
It seems that `input_shape=(4,))` is a typo? I'm not sure if this code will be rendered correctly. Could you adjust those lines as suggested above? (i.e. turn those lines to comments in the code example.

Reply:
I'm not sure about what you mean exactly: could you explain why `input_shape=(4,)` is a typo?If that's so then should it be changed too for `random_flax_module`?

Reply:
I think
```
prior={"bias": dist.Cauchy(), "kernel": dist.Normal()},input_shape=(4,))
```
should be changed to
```
prior={"bias": dist.Cauchy(), "kernel": dist.Normal()}
```

> should it be changed for random_flax_module too?

I think so. But it is better to put your note in the comment of the code, like
```
net = random_flax_module("net",
    flax.linen.Dense(features=1),
    # Alternatively, we can use a callable
    # prior=(lambda name, shape: dist.Cauchy() if name == "bias" else dist.Normal()),
    prior={"bias": dist.Cauchy(), "kernel": dist.Normal()},
    input_shape=(4,))
```